baseline.yaml

```diff
@@ -720,6 +720,55 @@ criteria:
       in a separate repository and fetched during
       a specific well-documented pipeline step.
     control_mappings: # TODO
+    security_insights_value: # TODO
+    scorecard_probe: # TODO
+  - id: OSPS-53
```

Comment:
I disagree with this one as worded fairly strongly. 

I would reword it "cryptographically attested" -- We've seen multiple issues with just signing a piece of software.

Reply:
@mlieberman85 If the desire is for more than signing— should we split this into Lv2 signature, and Lv3 for more complete attestation?

Reply:
I think that's reasonable. I think we should also be clear what just blanket "signing" should be interpreted as. Since people sign a lot of stuff for a lot of reasons, which is why folks have been moving to attestations so it's clear what the intent of the signing is.

Reply:
I think the first level should be plain signing with published singer identities. It's way easier to implement plain signing and it gets us initial integrity guarantees.